golang/cosmos/app/app.go

```diff
@@ -794,7 +794,21 @@ func upgrade10Handler(app *GaiaApp, targetUpgrade string) func(sdk.Context, upgr
 		app.VstorageKeeper.MigrateNoDataPlaceholders(ctx) // upgrade-10 only
 		normalizeProvisionAccount(ctx, app.AccountKeeper)
 
-		return app.mm.RunMigrations(ctx, app.configurator, fromVm)
+		mvm, err := app.mm.RunMigrations(ctx, app.configurator, fromVm)
```

Comment:
Check if err is non-nil here, and if so return. We don't want to proceed with swingset initialization in this case.